docs/user/alerting/stack-alerts/es-query.asciidoc

```diff
@@ -6,20 +6,20 @@ The ES query alert type is designed to run a user-configured {es} query over ind
 actions to run when the threshold condition is met.
 
 [float]
-==== Creating the alert
+==== Create the alert
 
-An ES query alert can be created from the *Create* button in the <<alert-management, alert management UI>>. Fill in the <<defining-alerts-general-details, general alert details>>, then select *ES query*.
+Fill in the <<defining-alerts-general-details, alert details>>, then select *ES query*.
 
 [role="screenshot"]
```

Comment:
I don't feel that the index is needed here.

Reply:
Are you suggesting to remove the screenshot here?